omero/sysadmins/system-requirements.txt

```diff
@@ -75,7 +75,7 @@ the user's HOME directory in :file:`$HOME/omero/tmp`,
 :file:`$HOME\\omero\\sessions`). If your home
```

Comment:
RFE: If an example Windows path is included it should also use the appropriate windows env-var (which appears to be either `%HOMEPATH%` or `%APPDATA%` depending on the system). Probably easier to just remove it.